### ISSUE TITLE
Device not recognized

### ISSUE THREAD

**Issue by vladdeda** [DarthAffe/RGBSyncPlus#10]
Any way you could impliment support for the Logitech G910 Orion Spark? It works great with my G502 Proteus spectrum. Just feels lacking without keyboard.

**Comment by DarthAffe** [comment]
the G910 is included and should work, but maybe there is a newer version of it out there. Could you check it's HID in the hardware-manager and report that back? I can add it if it's really missing.

**Comment by DarthAffe** [comment]
Hey, that's quite close but not exactly what I need - in then window on the right change to details and select Hardware-IDs in the dropdown - the values in the box below should look something like ```HID/BID_1111&PID_111&REV_0000``` or something like that - that's what I need.
You could also try it again with the latest version (updated it 2 days ago and it contains a newer version of the logitech-sdk which might also help)

**Comment by vladdeda** [comment]
![hwid](https://user-images.githubusercontent.com/19613372/51209542-15d72500-18d6-11e9-99fb-d7507dbb71cf.png)


I tried the new version. It makes my lights blink random colors. I think it might have something to do with the newest version of Aura Sync I installed as well. So I'm going to revert back to older version and see how that works out.

**Comment by vladdeda** [comment]
Yes, new version of rgbsync+ causes "christmas light" blinking effect. Also keyboard does not show as an available sync option. Mouse shows up to be added to sync list. Keyboard does not. Also corsair pump does not sync nor does the GPU.


![lights](https://user-images.githubusercontent.com/19613372/51210351-2ee0d580-18d8-11e9-9fe1-c66d5f9b7d76.png)
![keyboard](https://user-images.githubusercontent.com/19613372/51210352-2ee0d580-18d8-11e9-9e2a-8f321767d00e.png)
![lights 2](https://user-images.githubusercontent.com/19613372/51210353-2f796c00-18d8-11e9-8fc3-0694d3aed729.png)
